e2e/ChannelApeClient.spec.ts

```diff
@@ -684,4 +716,32 @@ describe('ChannelApe Client', () => {
     expect(channel.updatedAt.getUTCMilliseconds())
       .to.be.greaterThan(expectedCreatedAt.getUTCMilliseconds());
   }
+
+  /**
+   * Generates a random number based on the provided min and max values.
+   * @param {number} min Minimum number to generate (inclusive)
+   * @param {number} max Maximum number to generate (inclusive)
+   */
+  function randomNumber(min: number, max: number): number {
```

Comment:
We already have fake word generator. Please see the order creation test in this class.

Please remove this `randonmNumber` function and replace calls to this function with calls to the `faker` sdk we already have in place.